src/js/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = Object.freeze({
+  bootstrapVersion: '4.1.3'
```

Comment:
This is something I don't like personally at all. We should have a property in package.json and read it on build time.

Reply:
We use it in the build time too 😉 But we need it in source files too

Reply:
Sure, just don't hardcode it.